shuffle/package.json

```diff
@@ -45,7 +47,9 @@
     "react-share": "^4.4.1",
     "react-tooltip": "^5.13.1",
     "slugify": "^1.6.6",
+    "tailwind-merge": "^1.14.0",
     "tailwindcss": "3.3.2",
+    "tailwindcss-animate": "^1.0.6",
```

Comment:
I'm a bit sceptical if we need all of these... couldn't we add them later when we actually add a component that uses these?